src/codegeneration/SymbolTransformer.js

```diff
@@ -80,6 +82,8 @@ function isSafeTypeofString(tree) {
   return true;
 }
 
+var runtimeOption = parseStatement `$traceurRuntime.options.symbols = true`;
```

Comment:
I guess my concern is not well founded. I don't think it is reasonable to able to have different Scripts/Modules compiled with different options and expect them to work together.